ReleaseNotes.md

```diff
@@ -1 +1,3 @@
-
+Updates to existing cmdlets:
+- Add-PowerBIWorkspaceUser: allow adding service principals to workspaces.
+- Add-PowerBIWorkspaceUser: allow adding groups to workspaces.
```

Comment:
Both are same cmdlets. So we can have the changes in a single line?

Reply:
Good point - I've made this change.